sentry/templates/configmap-sentry.yaml

```diff
@@ -48,6 +48,9 @@ data:
     slack.client-id: "{{ .Values.slack.clientId }}"
     slack.client-secret: "{{ .Values.slack.clientSecret }}"
     slack.verification-token: "{{ .Values.slack.verificationToken }}"
+    {{- if .Values.slack.legacyApp }}
+    slack.legacy-app: "{{ .Values.slack.legacyApp }}"
```

Comment:
Thanks !
From what I see it should be a "python" boolean, not a string. Have you tried the behaviour on your instance?

Reply:
I haven't tried it in an instance. You're right, it should be a boolean here. I'll fix it.

Reply:
If you put `#   legacyApp: true` in the values
and then

```

 {{- if .Values.slack.legacyApp }}
    slack.legacy-app: True
{{- else }}
    slack.legacy-app: Flase
```

Reply:
Thanks for the good suggestions. fixed it.